scripts/pipeline/orthology_benchmark.py

```diff
@@ -221,6 +223,33 @@ def get_gtf_file(core_name: str, source_dir: str, target_dir: str) -> None:
         shutil.copy(gtf_file, target_dir)
 
 
+def prepare_gtf_files(core_names: List[str], source_dir: str, target_dir: str) -> None:
+    """Finds, copies and gunzips GTF files for specified core dbs.
+
+    Args:
+        core_names: Core db names.
+        source_dir: Path to the directory containing subdirectories with GTF files.
+            `production/ensemblftp` or MC's `nobackup/release_dumps`.
+        target_dir: Path to the directory where the GTF files will be copied.
+
```

Comment:
It's no harm to be clear that `target_dir` shouldn't contain extraneous gzipped GTF files:
```suggestion
        target_dir: Path to the directory where the GTF files will be copied. This
            directory should not contain any gzip-compressed GTF files that you do
            not intend to decompress.

```